subsys/usb/device_next/class/Kconfig.cdc_acm

```diff
@@ -21,6 +21,15 @@ config USBD_CDC_ACM_STACK_SIZE
 	help
 	  USB CDC ACM workqueue stack size.
 
+config USBD_CDC_ACM_ENSURE_SHORT_PKT_FOR_LAST_TX_PKT
+	bool "USB CDC ACM transmission always end with short packet"
+	help
+	  If the length of CDC ACM TX payload to be sent to USB Host
+	  is in multiple of max-packet-size of the BULK-IN endpoint,
+	  a Zero-Length Packet will be sent followig the last packet
```

Comment:
```suggestion
	  a Zero-Length Packet will be sent following the last packet
```

Reply:
Hi @ycsin, thanks for catching that. pushed a fix for this.